### DISCUSSION TITLE
Install ROCm on windows10

### DISCUSSION THREAD

**Discussion by xiaobo1025** [ROCm/ROCm#2744] [Q&A] [answered]
Friends, I have recently failed to install ROCm on my windows10 system. Can you give me a little help? I really thank you, my friends.

**Comment by zichguan-amd** [comment, marked as answer]
Hi @xiaobo1025, only a subset of ROCm is supported on Windows, please refer to the documentation [here](https://rocm.docs.amd.com/projects/install-on-windows/en/develop/) for a detailed guide on how to install and what components are available on Windows. You can also use WSL following the docs [here](https://rocm.docs.amd.com/projects/radeon/en/latest/docs/install/wsl/install-radeon.html).
